packages/vela/src/test-transactions.ts

```diff
@@ -98,7 +100,59 @@ const ETH_STOPLIMIT: TestParams<OptionsActionParams> = {
   },
 }
 
-export const passingTestCases = [
+export const MINT_VLP: TestParams<OptionsActionParams> = {
```

Comment:
Even though this is not being used, it should not be using OptionsParams. Might be best to just remove this one for now.